src/arrayMethodReduce.js

```diff
@@ -5,8 +5,16 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
-  };
-}
+    let accumulator = arguments.length >= 2 ? initialValue : this[0];
```

Comment:
```suggestion
    let accumulator = arguments.length === 2 ? initialValue : this[0];
```
Actually, there can be 2 arguments max